### PR TITLE
Tests: adding oversetRotCylinder test using Trilinos

### PR DESCRIPTION
**Pull request by lucbv**
**Pull-request type:**
- [ ] Bug fix 
- [ ] Documentation update
- [x] Feature enhancement

## Description of the pull-request
The current oversetRotCylinder is switched to use Trilinos/MueLu for the continuity solve.
The old test is renamed oversetRotCylinderHypre as is consistent with other tests in the regression suite.

## Checklist

*All pull requests*
- [x] Builds successfully (*must test on at least one system/compiler combination*)
  - Operating systems 
    - [x] Linux
    - [ ] MacOS
  - Compilers 
    - [x] GCC
    - [ ] LLVM/Clang
    - [ ] Intel compilers
    - [ ] NVIDIA CUDA
- [x] Compiles without warnings
- [x] Passes all unit tests
- [x] Passes all regression tests
- [x] Documentation builds without errors

*For new code updates*
- [ ] Documentation updates - additions to user, theory, and verification manuals
- [ ] New unit tests providing coverage for new code, bug fixes etc.
- [x] New regression tests exercising the new code

### CODE REVIEW

**Comment by lucbv** [comment]
This PR is related to issue #460

File: reg_tests/test_files/oversetRotCylinder/oversetRotCylinder.i [hunk 1]
```diff
@@ -15,13 +15,14 @@ linear_solvers:
...
-    preconditioner: boomerAMG
+    type: tpetra
+    method: gmres
+    preconditioner: muelu
     tolerance: 1e-5
     max_iterations: 200
     kspace: 5
-    output_level: 0
+    muelu_xml_file_name: ../../xml/milestone.xml
+    output_level: 1
```
Comment (sayerhs) [inline reg_tests/test_files/oversetRotCylinder/oversetRotCylinder.i]: Can we set the default `output_level` to 0?

Comment (lucbv) [reply reg_tests/test_files/oversetRotCylinder/oversetRotCylinder.i]: Sure

**Comment by lucbv** [comment]
Updated the input file to turn verbosity level down to 0 in continuity solve

**Pull request merged**